### PR TITLE
Jest tests

### PR DESCRIPTION
**Pull request by evansiroky**
add jest tests to this project

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -7,11 +7,13 @@
...
   "scripts": {
+    "cover": "mastarm test --coverage --coverage-paths index.js",
+    "lint": "mastarm lint",
     "prepublish": "mastarm prepublish index.js:build/index.js",
     "prestart": "yarn",
-    "test": "mastarm lint",
     "start": "mastarm build --serve example.js",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
+    "semantic-release": "semantic-release pre && npm publish && semantic-release post",
+    "test": "yarn run lint && mastarm test"
```
Comment (trevorgerhardt) [inline package.json:16]: `yarn run lint` is 13 characters, `mastarm lint` is 12. There really shouldn't be a need to add a lint command to npm scripts.

**Review by trevorgerhardt — CHANGES_REQUESTED** [review]
Just one note on npm scripts but the code looks good!

File: package.json (hunk 1 above)
Comment (evansiroky) [reply package.json:16]: The travis commands run lint standalone and then run the coverage script.

Comment (trevorgerhardt) [reply package.json:16]: If it's needed then 👌 I'm mainly just reminding that npm scripts are shortcuts.

**Review by trevorgerhardt — APPROVED** [review]

**Pull request merged**